### DISCUSSION TITLE
升级1.40版本后 无障碍打开后闪退

### DISCUSSION THREAD

**Discussion by wndlsj** [gkd-kit/gkd#211] [Q&A]
升级1.40版本后 无障碍打开后闪退 
安卓版本8.1.0 （小米6）
之前的1.30版本木问题

<img src="https://wx2.sinaimg.cn/mw690/69a84ad1ly1hjatuyw12oj20i20ucgmx.jpg">

<img src="https://wx2.sinaimg.cn/mw690/69a84ad1ly1hjatuyw1mej20i80wh75r.jpg">

**Comment by wndlsj** [comment]
已解决，感谢 [lisonge]

**Comment by lisonge** [comment]
https://github.com/gkd-kit/gkd/releases
